src/AbstractDatadirTestCase.php

```diff
@@ -116,8 +114,6 @@ protected function getTempDatadir(DatadirTestSpecificationInterface $specificati
     {
         $fs = new Filesystem();
 
-        $this->temp->initRunFolder();
```

Comment:
Proč?

Reply:
už to řeší php-temp https://github.com/keboola/php-temp/blob/master/src/Temp.php#L68